autoscheduler/scraper/serializers.py

```diff
@@ -36,3 +37,56 @@ def get_meetings(self, obj): # pylint: disable=no-self-use
             'end_time': meeting.end_time,
             'type': meeting.meeting_type,
         } for meeting in meetings]
+
+class TermSerializer(serializers.ModelSerializer):
+    """ Serializes a term into an object with information needed by /api/terms """
+    desc = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Department
+        fields = ['term', 'desc']
+
+
+    def get_desc(self, obj):
+        """ Uses term field to generate description for the term in the
+            form "Fall - College Station" format"""
+
+        def season_num_to_string(season_num):
+            """" Converts int representing season in 'term' field to a string to
+                use in get_term """
+            # put all translations here. Possibly inaccurate atm.
```

Comment:
```suggestion
            # Put all translations here. Possibly incomplete.
```
Single-line comments should all have a space after the pound sign and begin with a capital letter, I pointed out the others that should be changed